docs/src/components/ColorSwatchFamilyList/ColorSwatchFamilyList.component.tsx

```diff
@@ -0,0 +1,82 @@
+// REACT
+import React, { useState } from 'react';
+// VENDOR
+import styled from 'styled-components';
+import classNames from 'classnames';
+import { ColorSwatchFamily } from '../ColorSwatchFamily';
+// THEME
+import { colors, sizes } from '../../../../src/theme';
+
+
+// INTERFACES
+interface ColorSwatchFamilyListProps {
+  className?: string;
+}
+
+
+// STYLES
+const StyledColorSwatchFamilyList = styled.div`
+
```

Comment:
Doesn't look like we need this styled component.

Reply:
✅ I'll remove.

Reply:
I initially forgot to remove this, but I just removed it ✅